queries.sql

```diff
@@ -1,3 +1,9 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-12-31';
+SELECT name FROM animals WHERE neutered=true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name='Pikachu';
+SELECT * FROM animals WHERE neutered=true;
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+SELECT * FROM animals WHERE name NOT IN('Gabumon');
```

Comment:
- [ ] Great work writing most of the queries. Although you have a screenshot, you forgot to write this query in the `queries.sql` file:

> List names and escape attempts of animals that weigh more than 10.5kg.

Kindly include this query as it is part of the requirements for this milestone. 👍

Reply:
yeah i saw that and i added it in queries.sql line 6

Reply:
Hi @clin2on3mun, you just added it. Check the highlighted code that I have for the required change. 👍